include/odp/api/spec/ipsec.h

```diff
@@ -1626,6 +1696,21 @@ int odp_ipsec_sa_mtu_update(odp_ipsec_sa_t sa, uint32_t mtu);
  */
 void *odp_ipsec_sa_context(odp_ipsec_sa_t sa);
 
+/**
+ * Retrieve information about an IPSEC SA
+ *
+ * The pointers for cipher and auth key data in odp_ipsec_sa_param_t field will
+ * be set to NULL.
```

Comment:
Please, mention also that IP address pointers point to sa_info data.

Reply:
> 
> 
> > Let me know if the API spec looks OK now.
> 
> How should the other pointers than those for keys be set in the info structure? We discussed that the context pointer should be returned as is. That could be mentioned. But what about the tunnel address and lookup address pointers? Should they be returned as they were in the SA creation call (this is how I understand the current API, although ODP cannot know if the pointers are still valid) or should they point to the arrays that are part of the info structure or should they be set to null or should they have undefined values? I would not like the first alternative as it would require ODP to remember the original pointer values even though in many applications the values would be stale.

V6: Updated the spec saying:
1. context pointer is returned as it was programmed during sa creation
2. all the IP address pointers points to the corresponding buffers available in the SA info structure.

> 
> 
> Please, mention also that IP address pointers point to sa_info data.

Done.